pkg/client/auth/oidc/device.go

```diff
@@ -48,7 +48,12 @@ func AuthenticateDevice(config DeviceDetails) (*TokenCredentials, error) {
 		if err != nil {
 			return nil, err
 		}
-		defer cmd.Process.Kill()
+		defer func() {
+			err := cmd.Process.Kill()
+			if err != nil {
+				fmt.Printf("Error killing your process: %s", err)
```

Comment:
Same here.  Should probably be logging these as warnings and not Printf?

Reply:
Agreed. This should be a log.
And please start error messages with lower-case letters.
Since they get embedded in sequences of errors like `error 1: error 2: error 3`.